test/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
this is what we're doing in `near/near-evm` which makes it easier to use/deploy web3 contracts, as opposed to using truffle contracts

Reply:
cool beans